08_advanced/memory_snapshots/sdxl.py

```diff
@@ -0,0 +1,79 @@
+# # Memory snapshots for Stable Diffusion XL
+#
+# We will load Stable Diffusion XL 1.0 in CPU memory then take a memory snapshot
+# to improve cold boot times.
+
+import time
+import modal
+
+
+MODEL_ID: str = "stabilityai/stable-diffusion-xl-base-1.0"
+CACHE_PATH: str = "/vol/cache"
+
+
+image = (
+    modal.Image.debian_slim()
+    .pip_install(
+        "torch",
```

Comment:
Also these deps should be `==` pinned. Ref _Review of synmon on Examples_ in our Notion.